### PR TITLE
Adding new Example Sklearn Heart Attack Predictor

### PR DESCRIPTION
**Pull request by lakshikaparihar**
## What changes are proposed in this pull request?

Adding Scikit-learn Example for Heart Rate Prediction

## How is this patch tested?

This code is tested locally using mlflow CLI

## Does this PR change the documentation?

- [ ] No. You can skip the rest of this section.
- [x] Yes. Make sure the changed pages / sections render correctly by following the steps below.

1. Check the status of the `ci/circleci: build_doc` check. If it's successful, proceed to the
   next step, otherwise fix it.
2. Click `Details` on the right to open the job page of CircleCI.
3. Click the `Artifacts` tab.
4. Click `docs/build/html/index.html`.
5. Find the changed pages / sections and make sure they render correctly.

### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?
Components 
- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [x] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface 
- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language 
- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations
- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>
### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Comment by Bidek56** [comment]
@lakshikaparihar Are you sure you have committed the correct code?
This PR has 206 commits and changes 584 files. Something seems odd.

File: examples/heart_attack_predictor/log_model.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Importing Libraries
```
Comment (harupy) [inline examples/heart_attack_predictor/log_model.py:1]: How does this example differ from the existing sklearn examples?

**Comment by lakshikaparihar** [comment]
> @lakshikaparihar Are you sure you have committed the correct code? This PR has 206 commits and changes 584 files. Something seems odd.

Seems odd to me as well. Initially, when I created the PR it had only 6 file changes but as soon as the workflow started, it showed 584 files changed.

File: examples/heart_attack_predictor/log_model.py (hunk 1 above)
Comment (lakshikaparihar) [reply examples/heart_attack_predictor/log_model.py:1]: The existing examples are good but when I was exploring MLflow, I got stuck at many points like
How to log a pre-trained model (all the existing examples have added the training code with it)
How to add signature in your model
How to register your model through code
How to shift your model into production through code

So this example clarifies all the above points

**Comment by Bidek56** [comment]
@lakshikaparihar  Did you start with an old fork that got rebased from master? I would suggest:
1. Closing this pr 
2. Creating a new fork 
3. Creating a new PR

**Comment by lakshikaparihar** [comment]
> @lakshikaparihar Did you start with an old fork that got rebased from master? I would suggest:
> 
> 1. Closing this pr
> 2. Creating a new fork
> 3. Creating a new PR

I did fetch the new changes in my forked repo before creating the PR but might have missed some steps, so let me try this.

**Pull request closed without merging**